lib.ftd

```diff
@@ -5586,3 +5586,174 @@ children: $page-wrapper.child
 -- end: ftd.column
 
 -- end: page-wrapper
+
+
+
+
+
+
+
+
+
+
+-- component new-article:
+caption title:
+string date:
+string article-length:
+body body:
+ftd.image-src src:
+ftd.image-src paperclip:
+children child:
+
+-- ftd.column:
+width: fill-container
+align-content: center
+margin-top.px: 84
+margin-bottom.px: 200
+
+-- ftd.row:
+width: fill-container
+max-width.fixed.px: 1145
+spacing: space-between
+
+-- ftd.column:
+width.fixed.percent: 70
+
+-- ftd.column:
+background.solid: white
```

Comment:
```suggestion
-- ftd.column:
width.fixed.percent: 70
background.solid: white
```

Reply:
Can't do this change cause the background changes...for `read next >` text at the bottom right